Plugins/TGeo/include/Acts/Plugins/TGeo/TGeoLayerBuilder.hpp

```diff
@@ -74,7 +74,7 @@ class TGeoLayerBuilder : public ILayerBuilder {
         : volumeName(""),
           sensorNames({}),
           localAxes("XZY"),
-          envelope(std::pair<double, double>(1_mm, 1_mm)) {}
+          envelope(std::pair<double, double>(0_mm, 0_mm)) {}
```

Comment:
Maybe your issue is due to the fact two layer are closer than 1mm ? I think the initialisation should be kept to (1,1) but maybe @asalzburger can comment since he has used the TGeo way more than I did.

Reply:
Yes it could be.. I changed it back to the original 1 mm envelope.

Reply:
Yea, it needs some sort of an envelope, I guess - but since you have changed already, seems ok.